Assets/stylesheet.css

```diff
@@ -0,0 +1,131 @@
+ /* *{
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+} */
+nav{
+    position: absolute;
+    left: 160px;
+    top: 85px;
+    width: 980px;
+    height: 36px;
+    display: inline-block;
+    border: #7a7a7a 1px solid;
+    border-radius: 4px;
+    background: linear-gradient(#737373 50%,#777777 25%,#5d5d5d 50% );
+}
+nav ul{
+    height: 36px;
+    padding: 0;
+    margin-top: 0;
+}
+nav a{
+    text-shadow: 2px 2px #363636;
+    text-decoration: none;
+    color: white;
+}
+nav li{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    list-style-type: none;
+    float: left;
+    width: 106px;
+    height: 36px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    font-size: 12px;
+    color: white;
+    border-left: 1px solid #464646;
+}
+nav li i{
+    padding: 0 4px;
+}
+
+input{
+    height: 19.5px;
+    width: 85px;
+    border-radius: 50px;
+    box-shadow: 3px 1px #3c3c3c;
+    background: inherit;
+}
+main{    
+    position: relative;
+    background-image: url(./main_background.jpg);
+    background-position: 50% 100%;
+    background-repeat: no-repeat;
+    height: 538px;
+    width: 1268px;
```

Comment:
`SUGGESTION`

Let's reposition the background image to fill the page as shown below:

|Current|Preffered|
| :-----: | :------: |
| ![appl com current](https://user-images.githubusercontent.com/36057474/62105869-899db580-b29b-11e9-9c97-ff6996ac5432.png)  |  ![preffered app](https://user-images.githubusercontent.com/36057474/62092488-488eac80-b26d-11e9-82c8-95cf194524fe.png) |